extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -69,6 +69,23 @@ export let InteractiveComponentDef;
  */
 export let InteractiveReactData;
 
+/**
+ * @typedef {{
+ *   brand-favicon: ?string,
+ *   brand-label: string,
+ *   cta-text: ?number,
```

Comment:
Why it this a number?

Reply:
from the [I2I](https://github.com/ampproject/amphtml/issues/36460):
`cta-text: 1, // Optional. Number. Defaults to “Buy now”. Keys to a CTA text option for i18n.`
I wonder if we need to add comment about this.